libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -68,6 +68,14 @@ const AP_Param::GroupInfo AC_Avoid::var_info[] = {
     // @User: Standard
     AP_GROUPINFO_FRAME("BEHAVE", 5, AC_Avoid, _behavior, AP_AVOID_BEHAVE_DEFAULT, AP_PARAM_FRAME_COPTER | AP_PARAM_FRAME_HELI | AP_PARAM_FRAME_TRICOPTER),
 
+    // @Param: BACKUP_SPD
+    // @DisplayName: Maximum back away speed
+    // @Description: Maximum speed that will be used to back away from obstacles in GPS modes (m/s). Set zero to disable
+    // @Units: m/s
+    // @Range: 0 2
+    // @User: Standard
+    AP_GROUPINFO("BACKUP_SPD", 6, AC_Avoid, _backaway_max_spd, 0.5f),
```

Comment:
Sorry to be a little pedantic but let's be consistent about whether it's "backaway" or "backup".  So I think the parameter is called "BACKUP_" because this is shorter so let's make all the internal variables and descriptions "backup" as well.

Also while we're at it, let's rename _backaway_max_spd to "backup_speed_max".  I prefer to have the "_max" at the end... this wouldn't be a blocker but if we're renaming it anyway..